Marlin/Marlin_main.cpp

```diff
@@ -4982,32 +4988,26 @@ inline void gcode_G28() {
 
   #if ENABLED(DELTA_AUTO_CALIBRATION)
     /**
-     * G33: Delta '4-point' auto calibration iteration
-     *
-     * Usage: G33 <Cn> <Vn>
-     *
-     *  C  (default) = Calibrate endstops, height and delta radius
-     *
-     *  -2, 1-4: n x n probe points, default 3 x 3
-     *
-     *    1: probe center
-     *       set height only - useful when z_offset is changed
-     *    2: probe center and towers
-     *       solve one '4 point' calibration
-     *   -2: probe center and opposite the towers
-     *       solve one '4 point' calibration
-     *    3: probe 3 center points, towers and opposite-towers
-     *       averages between 2 '4 point' calibrations
-     *    4: probe 4 center points, towers, opposite-towers and itermediate points
-     *       averages between 4 '4 point' calibrations
-     *
-     *  V  Verbose level (0-3, default 1)
-     *
-     *    0: Dry-run mode: no calibration
-     *    1: Settings
-     *    2: Setting + probe results
-     *    3: Expert mode: setting + iteration factors (see Configuration_adv.h)
-     *       This prematurely stops the iteration process when factors are found
+     * G33 - Delta '4-7-point' auto calibration (Requires DELTA)
+     * 
+     * Usage:
+     *   G33 <Cn> <Vn>
+     *   
+     *     Cn = n=-7 -> +7 : n*n probe points
+     *          calibrates height ('1 point'), endstops, and delta radius ('4 points') 
+     *          and calibrates tower angles with n >= 3 ('7+ points')
+     *          n=0  <default>
+     *          n=1  probes center / sets height only
+     *          n=-1 same but 1 iteration only
+     *          n=2  probes center and towers / sets height, endstops and delta radius
+     *          n=-2 same but opposite towers
+     *          n=3  probes all points: center, towers and opposite towers / sets all
+     *          n>3  probes all points multiple times and averages
+     *          n<=3 same but tower angle calibration disabled
```

Comment:
Like this?

    /**
     * G33 - Delta '4-7-point' auto calibration (Requires DELTA)
     * 
     * Usage:
     *   G33 <Pn> <A> <O> <T> <Vn>
     *   
     *     Pn = n=-7 -> +7 : n*n probe points
     *          calibrates height ('1 point'), endstops, and delta radius ('4 points') 
     *          and tower angles with ('7+ points')
     *          n=1  probes center / sets height only
     *          n=2  probes center and towers / sets height, end-stops and delta radius
     *          n=3  probes all points: center, towers and opposite towers / sets all
     *          n>3  probes all points multiple times and averages
     *     A  = abort 1 point delta height calibration after 1 probe
     *     O  = use oposite tower points instead of tower points with 4 point calibration
     *     T  = do not calibrate tower angles with 7+ point calibration
     *     Vn = verbose level (n=0-2 default 1)
     *          n=0 dry-run mode: no calibration
     *          n=1 settings 
     *          n=2 setting + probe results 
     */

Is already adapted like this